test/corpus/pipelines.txt

```diff
@@ -35,6 +35,21 @@ Function with arguments
                 (field_identifier))
             (int_literal))))
 
+====================================
+Invalid Function
```

Comment:
Should be called Invalid function parameters.

Also the same thing with with variable definition could be added:

`{{ print $var:=5 }}`